### ISSUE TITLE
UI: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#949]
When clicking on the Show Granules link in the collection page, we're seeing `Uncaught RangeError: Maximum call stack size exceeded at Array.sort`. It's not every time, but frequently enough to be a concern.

**Comment by StrayCat1** [comment]
```
is_arguments.js?a229:9 Uncaught RangeError: Maximum call stack size exceeded
The above error occurred in the <HelmetWrapper> component:
    in HelmetWrapper (created by Meta)
    in Meta (created by DescriptionView)
    in div (created by DescriptionView)
    in DescriptionView (created by Detail)
    in div (created by Detail)
    in div (created by Detail)
    in Detail (created by Connect(Detail))
    in Connect(Detail) (created by Route)
    in Route (created by withRouter(Connect(Detail)))
    in withRouter(Connect(Detail)) (created by Root)
    in div (created by Root)
    in Route (created by Root)
    in Switch (created by Root)
    in div (created by Root)
    in main (created by Middle)
    in Middle (created by Content)
    in div (created by FlexRow)
    in FlexRow (created by Content)
    in Content (created by Layout)
    in div (created by FlexColumn)
    in FlexColumn (created by Layout)
    in div (created by Background)
    in div (created by Background)
    in Background (created by Layout)
    in Layout (created by Root)
    in Root (created by Connect(Root))
    in Connect(Root) (created by Route)
    in Route (created by withRouter(Connect(Root)))
    in withRouter(Connect(Root))
    in Switch
    in div
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Connect(ConnectedRouter))
    in Connect(ConnectedRouter)
    in Provider
```
This appears to be an issue with react-helmet-
https://github.com/nfl/react-helmet/issues/373

**Comment by StrayCat1** [comment]
This is fixed in v6.0.0-beta. We either have to fork v5.2 and fix ourselves or wait for a stable release.
